sentry/src/main/java/io/sentry/protocol/User.java

```diff
@@ -232,30 +232,6 @@ public void setIpAddress(final @Nullable String ipAddress) {
     this.ipAddress = ipAddress;
   }
 
-  /**
```

Comment:
Can we also get rid of JsonKeys.OTHER here?

Reply:
i guess it's fine, as it was deprecated 2 years ago